specification/security/resource-manager/Microsoft.Security/preview/2023-08-01-preview/examples/Pricings/GetPricingByNameStorageAccounts_example.json

```diff
@@ -0,0 +1,37 @@
+{
+  "parameters": {
+    "api-version": "2023-08-01-preview",
+    "scopeId": "subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23",
+    "pricingName": "StorageAccounts"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/providers/Microsoft.Security/pricings/StorageAccounts",
+        "name": "StorageAccounts",
+        "type": "Microsoft.Security/pricings",
+        "properties": {
+          "pricingTier": "Standard",
+          "isEnabled": true,
+          "freeTrialRemainingTime": "PT0S",
+          "enablementTime": "2023-03-01T12:42:42.1921106Z",
+          "subPlan": "PerStorageAccount",
```

Comment:
Please update to DefenderForStorageV2, sub plan PerStorageAccount doesn't support extensions.

Reply:
Update also in specification/security/resource-manager/Microsoft.Security/preview/2023-08-01-preview/examples/Pricings/ListPricingsWithResourceFilter_example.json file